source/_integrations/reolink.markdown

```diff
@@ -168,6 +169,10 @@ Depending on the supported features of the camera, select entities are added for
 - Auto track method (Digital, Digital first, Pan/Tilt first)
 - Doorbell LED (Stay off, Auto, Auto & always on at night)
 - HDR* (Off, On, Auto)
+- Chime play ringtone
```

Comment:
what does the "loop" option do? loop over a default ringtone? Or is Loop the name of a ringtone?

Reply:
Loop is just the name of a ringtone.
I did not think of the names, they come from the Reolink app